server/server_test.go

```diff
@@ -91,6 +96,63 @@ func TestServerCall(t *testing.T) {
 			}
 		}
 	})
+	t.Run("Unimplemented hook", func(t *testing.T) {
```

Comment:
Just FYI, the testing framework will replace whitespaces with underscores in test names.  This doesn't actually break anything -- just thought I'd mention it.